src/server/gc_worker/compaction_filter.rs

```diff
@@ -141,8 +175,7 @@ impl WriteCompactionFilter {
         if self.write_batch.count() > DEFAULT_DELETE_BATCH_COUNT {
             let mut opts = WriteOptions::new();
             opts.set_sync(false);
-            let raw_batch = self.write_batch.as_inner();
-            self.db.write_opt(raw_batch, &opts).unwrap();
+            self.engine.write_opt(&self.write_batch, &opts).unwrap();
```

Comment:
The limit of write_batch is too large. A large write_batch may block write operation in other threads too long.

Reply:
You mean `128` is too large?

Reply:
Sorry, I read it wrong.